src/dao/MoodDao.java

```diff
@@ -15,6 +19,12 @@
 public class MoodDao {
     
     private final ConnectionManager cm;
```

Comment:
It doesn't seem like you're using this so you could remove it. The thing it's helpful for here is to get the name of the data directory but it's fine to have it in a string like you are doing. If you want to do that you don't need to make a variable because it has static fields; you can just get the data directory with `ConnectionManager.DATA_DIR`. Either way's fine, and just removing the variable is probably easier.